#!/usr/bin/env python
PACKAGE = "traxxas"


from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Min_Speed",double_t, 0,"Min Speed",1.245, 0, 2)
gen.add("Max_Speed",double_t, 0,"Max Speed",  5 , 2, 10)

exit(gen.generate(PACKAGE, "traxxas","motor"))